### DISCUSSION TITLE
Volar - disble linewidth on format vue template

### DISCUSSION THREAD

**Discussion by Pedroprt** [quasarframework/quasar#14707] [General - Components / Directives / etc] [answered]
My VSCode is line breaking the template around 80th column.
I unistalled Prettier and it still breaks the lines.
When I unistalled Volar the lines stopped to break. So I think that config is in volar.
On Volar settings I can't find nothing like 'linewidth'.
How can I disable the auto-linebreak on formatting templates ?

**Comment by MilosPaunovic** [comment, marked as answer]
It might have to do with default line wrapping settings of  `VSCode`. Try adding this in the `settings.json` file:

```json
  "editor.wordWrapColumn": 100,
  "html.format.wrapLineLength": 100,
```

But as this is not related to Quasar at all, try to ask this kinds of questions on the appropriate repositories and/or Google.

**Comment by Pedroprt** [comment]
Thanks. It worked.
I Googled it for a week and found nothing.
